src/packages/cli/src/Init.ts

```diff
@@ -252,6 +258,11 @@ export class Init implements Command {
       }
     }
 
+    fs.writeFileSync(
+      path.join(outputDir, ".gitignore"),
+      defaultGitIgnore()
+    );
+
```

Comment:
I think we should try catch this, so that if it fails it doesn't stop.
I would consider writing the .gitignore optional/not required, if we can write it that's ok, if not it's also ok.